lib/openstack-api/index.js

```diff
@@ -80,11 +84,15 @@ exports.launchLedgerNode = async ({
   return testCloud.createServer({auth, cloudConfig, image, name});
 };
 
-exports.launchOperationGenerator = async ({auth}) => {
+exports.launchOperationGenerator = async ({auth, dockerComposeB64}) => {
   const cloudConfig = fs.readFileSync(
     path.join(__dirname, 'cloud-config-operation-generator.yml'), 'utf8');
+  const instanceConfig = 'PUBLIC_IP={{ ds.ec2_metadata.local_ipv4 }}';
+  cloudConfig = cloudConfig.replace('_ENVFILE_', instanceConfig);
+  cloudConfig = cloudConfig.replace('_DOCKER_COMPOSE_', dockerComposeB64);
   const name = `operation-generator-${uuid()}`;
   const image = '27395632-3a9f-4a62-803a-c5cf0b702ba8';
```

Comment:
I know we're presently having difficulty with the Ubuntu 18.04 base image, but that's what we would want to use here just like the ledger nodes.